src/main/java/racingcar/CarPosition.java

```diff
@@ -0,0 +1,26 @@
+package racingcar;
+
+public class CarPosition {
+
+    public static int DEFAULT_POSITION = 0;
+    private int position;
+
+    private CarPosition(int position) {
+        validate(position);
+        this.position = position;
+    }
+
+    public static CarPosition of(int position) {
+        return new CarPosition(position);
+    }
+
```

Comment:
DEFAULT_POSITION으로 객체 생성하는 정적 팩토리 메소드를 하나 만드는건 어떨까요?
```suggestion
    public static CarPosition newInstacne() {
        return new CarPosition(DEFAULT_POSITION);
    }
```